packages/wasm-dpp/test/unit/decodeProtocolEntityFactory.spec.js

```diff
@@ -4,7 +4,10 @@ const { encode } = require('@dashevo/dpp/lib/util/serializer');
 let { decodeProtocolEntity, ProtocolVersionParsingError, SerializedObjectParsingError } = require('../..');
 const { default: loadWasmDpp } = require('../..');
 
-describe('decodeProtocolEntityFactory', () => {
+// TODO: decodeProtocolEntity was broken after serialization refactoring
+//   it was mostly used in js-drive and does not seem to be needed anymore
+//   can we remove it completely?
+describe.skip('decodeProtocolEntityFactory', () => {
```

Comment:
Do we plan to bring this function back in the future? If not, maybe it's better to just delete it?

Reply:
I plan to get back to it one more time in the next iteration where I will be fixing SDK. There's one place in the wallet-lib that uses this function, so I'll remove it completely alongside with the bindings.

Reply:
Okay, makes sense 👍